nodes/nodes/nodes-nodes-adding-node-iso.adoc

```diff
@@ -24,6 +29,16 @@ The following platforms are supported for this method of adding nodes:
 * `none`
 * `external`
 
+[id="supported-architectures_{context}"]
+== Supported architectures
+
+The following architecture combinations have been validated to work using this process:
```

Comment:
Maybe some food for thought. All good either way. 

```suggestion
The following architecture combinations have been validated to work when using this process:
```

and for some overkill:

```suggestion
The following architecture combinations have been validated to work when adding worker nodes to an on-premise cluster:
```

or maybe even the following, which parallels "Supported platforms" above it. 

```suggestion
The following architecture combinations have been validated to work when adding worker nodes:
```

Reply:
I think I like the last one to keep the phrasing parallel, thanks

Reply:
That said I think I want to also keep the "when using this process" or something like "this method of" to specify that I'm talking about this particular way of adding nodes (since I think there is at least one other method with different limitations)